### PR TITLE
(#37) [Improvement]: sqlite database removed from repository

### LINKED ISSUE #37: Remove sqlite database from repo
**Is your feature request related to a problem? Please describe.**
A sample sqlite database is committed in the repo.

**Describe the solution you'd like**
* Delete it
* Add initial fixtures in json format
* Update readme explain how import this fixture
* Modify settings allowing multiple databases getting from environment variables.

**Describe alternatives you've considered**
Nope

**Additional context**
Nope

### PR DESCRIPTION
**Pull request by francescarpi**
## Description

Fix  #37 

This PR remove sqlite database from repository, add demo fixtures and update documentation explain how to use them.

## Code formating

The code has to be well formatted following the formatting rules. For example in Python the code has to follow the PEP8.

### CODE REVIEW

**Comment by francescarpi** [comment]
I keep it in draft mode waiting #51 be deployed.

File: .env.dev.sample [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ; To use dev env ini file.
 ; cp .env.dev.sample .env
-DEBUG=on
\ No newline at end of file
+DEBUG=on
```
Comment (jbagot) [inline .env.dev.sample:3]: DEBUG=True

**Review by jbagot — CHANGES_REQUESTED** [review]

File: .env.dev.sample (hunk 1 above)
Comment (francescarpi) [reply .env.dev.sample:3]: Ok, I'll change it tomorrow.
Thanks!

File: .env.dev.sample [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+DATABASE_URL=sqlite:///anem-per-feina.db
```
Comment (XaviTorello) [inline .env.dev.sample:4]: Maybe it's more clear if we preserve the `.sqlite` extension. I know that `.db` also is normally interpreted as sqlite, but maybe it's simpler and avoids misunderstandings 🤷🏻‍♂

**Review by XaviTorello — CHANGES_REQUESTED** [review]
Thanks @francescarpi once DEBUG boolean is fixed this will be fine :)

The sqlite vs db extension is not a core decision, so it's up to you :)

**Comment by francescarpi** [comment]
Hi!
I did it based on the official documentation

https://django-environ.readthedocs.io/en/latest/

But its more cleared now, with your proposals.

Changes done!
Thanks!

**Review by XaviTorello — APPROVED** [review]
Looks good, many thanks @francescarpi  :beers:

File: jobs/settings/__init__.py [hunk 3]
```diff
@@ -64,10 +64,7 @@
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    },
+    'default': env.db(),
     # 'default': {
     #     'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (fullonic) [inline jobs/settings/__init__.py:69]: @francescarpi We can also remove this commented code about postgres database.

**Pull request merged**